### DISCUSSION TITLE
Welcome to Day 2

### DISCUSSION THREAD

**Discussion by Patecatl848** [community/events#61] [Community]
Another fantastic start, behind the scenes; rundown on Day 1; and an Introduction of the Day with Thomas.

**Comment by imjohnbo** [comment]
👋 Welcome and yay Day 2! What is everyone [looking forward to today](https://github.com/community/events/discussions/58)?

**Reply by ghostinhershell** [reply]
Currently watching the Spotlight on Maintainers w/@abbycabs and loving it!

**Reply by jmmccota** [reply]
:rocket:

**Comment by bu7maydAbdu** [comment]
quantum development , wiiiiiiiiiiiiiiiiild 🔥

**Comment by emirkmo** [comment]
Anyone know of a chat room with live discussion? Very cool stuff honestly. Thanks for making this available for free.

**Comment by smmdsa** [comment]
cheers! thx for sharing!
